docs/docusaurus/docs/core/define_expectations/create_an_expectation.md

```diff
@@ -32,15 +32,15 @@ An Expectation is a verifiable assertion about your data. Expectations make impl
 
 1. Choose an Expectation to create.
 
-   GX comes with many built in Expectations to cover your data quality needs.  You can find a catalog of these Expectations in the [Expectation Gallery](https://greatexpectations.io/expectations/).  When browsing the Expectation Gallery you can filter the available Expectations by the data quality issue they address and by the Data Sources they support.  There is also a search bar that will let you filter Expectations by matching text in their name or docstring.
+   GX comes with many built in Expectations to cover your data quality needs.  You can find a catalog of these Expectations in the [Expectation Gallery](https://greatexpectations.io/expectations/).  When browsing the Expectation Gallery you can filter the available Expectations by the data quality issue they address and by the Data Sources they support.  There is also a search bar that will let you filter Expectations by matching text in their name or description.
```

Comment:
**note for reviewer:** "docstring" seemed like internal jargon so I changed this to more user-perspective language.

Reply:
I think either way reads fine, but FYI `docstring` isn't internal to GX - it's a python concept. But if you're looking at it from the perspective of the gallery, I think I agree that it's an implementation detail.